Makefile

```diff
@@ -105,4 +105,5 @@ verify-go-mod: ## Verify the go modules
 dockerbuild: ## Runs docker build
 	$(call ndef, GITHUB_AUTH_TOKEN)
 	docker build . --file Dockerfile --tag $(IMAGE_NAME) 
+	docker build . --file Dockerfile.gsutil --tag $(IMAGE_NAME)-gsutil
```

Comment:
Why do we need both Dockerfile and Dockerfile.gsutil commands, ideally just Dockerfile should be sufficient ?

Reply:
The `gsutil` is required by the `cron` to run and copy the images to `GCS`. 

We don't want the base image to contain `gstuil`  which is unnecessary, increases the chances of vulnerabilities.

Reply:
another image seems redundant just for gsutil, it is pretty trusted code, can keep in same. upto you.